File: app/src/main/java/com/example/whattodo/model/Event.java
package com.example.whattodo.model;

public class Event {

    private String Titol, Localitzacio, Telefon, Detalls, Adreça, Ciutat, Qualificacio, URL;
    private Double Latitud, Longitud;

    public Event(){}

    public Event(String titol, String localitzacio, String telefon, String detalls, String adreça, String ciutat, Double latitud, Double longitud, String qualificacio, String url){
        this.Titol = titol;
        this.Localitzacio = localitzacio;
        this.Telefon = telefon;
        this.Detalls = detalls;
        this.Adreça = adreça;
        this.Ciutat = ciutat;
        this.Latitud = latitud;
        this.Longitud = longitud;
        this.Qualificacio = qualificacio;
        this.URL = url;
    }

    public String getTitol(){
        return Titol;
    }

    public String getLocalitzacio(){
        return Localitzacio;
    }

    public String getTelefon(){
        return Telefon;
    }

    public String getDetalls(){
        return Detalls;
    }

    public String getAdreça(){
        return Adreça;
    }

    public String getCiutat() { return Ciutat; }

    public Double getLatitud(){
        return Latitud;
    }

    public Double getLongitud() { return Longitud; }

    public String getQualificacio(){
        return Qualificacio;
    }

    public String getURL(){return URL;}

}

File: app/src/main/java/com/example/whattodo/model/Event.kt
package com.example.whattodo.model

class Event {
    var Titol: String? = null
        private set
    var Localitzacio: String? = null
        private set
    var Telefon: String? = null
        private set
    var Detalls: String? = null
        private set
    var Adreça: String? = null
        private set
    var Ciutat: String? = null
        private set
    var Qualificacio: String? = null
        private set
    var Latitud: Double? = null
        private set
    var Longitud: Double? = null
        private set
    var URL: String? = null
        private set

    constructor() {}
    constructor(Titol: String?, Localitzacio: String?, Telefon: String?, Detalls: String?, Adreça: String?, Ciutat: String?, Latitud: Double?, Longitud: Double?, Qualificacio: String?, URL: String?) {
        this.Titol = Titol
        this.Localitzacio = Localitzacio
        this.Telefon = Telefon
        this.Detalls = Detalls
        this.Adreça = Adreça
        this.Ciutat = Ciutat
        this.Latitud = Latitud
        this.Longitud = Longitud
        this.Qualificacio = Qualificacio
        this.URL = URL
    }
}